i18n/translation-en.json

```diff
@@ -252,6 +254,13 @@
   "tool": {
     "pan": "Pan",
     "select": "Select",
-    "Marquee": "Marquee Zoom"
+    "Marquee": "Marquee Zoom",
+    "Link": "Link URL or Page"
+  },
+  "link": {
+    "URL": "URL",
+    "Page": "Page",
```

Comment:
to make things consistent can we make them lowercased just as other keys?

Reply:
Fixed